packages/esm-appointments-app/src/appointments/appointment-tabs.test.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { openmrsFetch } from '@openmrs/esm-framework';
-import { renderWithSwr, waitForLoadingToFinish } from '../../../../tools/test-helpers';
+import { renderWithSwr, waitForLoadingToFinish } from 'tools';
+import { mockAppointmentsData } from '__mocks__';
 import AppointmentTabs from './appointment-tabs.component';
-import { mockAppointmentsData } from '../../../../__mocks__/appointments.mock';
 
 const mockOpenmrsFetch = openmrsFetch as jest.Mock;
 
-jest.setTimeout(15000);
+jest.setTimeout(10000);
 
 describe('AppointmentTabs', () => {
   xit(`renders tabs showing different appointment lists`, async () => {
```

Comment:
We should ticket restoring this test.

Reply:
https://openmrs.atlassian.net/browse/O3-2682